# Introdução à Ciência de Dados - UFPB
Professor: Yuri Malheiros  
Aluno: João Victor Alcoforado de Araújo  
Matrícula: 20180083830

## Medidas de Centralidade e Dispersão

A partir do dataset `movies.csv` calcule a média, mediana e desvio padrão para as colunas:

In [5]:
import pandas as pd
movies_df = pd.read_csv('https://raw.githubusercontent.com/yurimalheiros/icd/master/datasets/movies.csv')
movies_df = movies_df[['Title', 'US Gross', 'Production Budget', 'Running Time (min)', 'IMDB Rating', 'IMDB Votes']]
movies_df.head()

Title   US Gross  Production Budget  \
0              The Land Girls   146083.0          8000000.0   
1      First Love, Last Rites    10876.0           300000.0   
2  I Married a Strange Person   203134.0           250000.0   
3        Let's Talk About Sex   373615.0           300000.0   
4                        Slam  1009819.0          1000000.0   

   Running Time (min)  IMDB Rating  IMDB Votes  
0                 NaN          6.1      1071.0  
1                 NaN          6.9       207.0  
2                 NaN          6.8       865.0  
3                 NaN          NaN         NaN  
4                 NaN          3.4       165.0

In [14]:
def attribute_resume(series):
    print(series.name)
    print(f'Mean = {series.mean()}')
    print(f'Median = {series.median()}')
    print(f'Standard Deviation = {series.std()}')

### US Gross

In [17]:
attribute_resume(movies_df['US Gross'])

US Gross
Mean = 44002085.163744524
Median = 22019465.5
Standard Deviation = 62555311.39066204


### Production Budget

In [18]:
attribute_resume(movies_df['Production Budget'])

Production Budget
Mean = 31069171.4484375
Median = 20000000.0
Standard Deviation = 35585913.44464446


### Running Time

In [19]:
attribute_resume(movies_df['Running Time (min)'])

Running Time (min)
Mean = 110.19354838709677
Median = 107.0
Standard Deviation = 20.171013677535424


### IMDB Rating

In [20]:
attribute_resume(movies_df['IMDB Rating'])

IMDB Rating
Mean = 6.283467202141896
Median = 6.4
Standard Deviation = 1.2522899386004838


### IMDB Votes

In [21]:
attribute_resume(movies_df['IMDB Votes'])

IMDB Votes
Mean = 29908.64457831325
Median = 15106.0
Standard Deviation = 44937.58233504965


___
### Análise dos Resultados

Quais colunas tiveram valores muito diferentes entre média e mediana? Quais os seus desvios padrão?

    As colunas US Gross, Production Budget e IMDB Votes, seus desvios padrões são respectivamente 62555311, 35585913 e 44937.

Que conclusões podemos tirar desses resultados?

    A primeira delas é que os valores das colunas US Gross, Production Budget e IMDB Votes são discrepantes. Portanto, pode-se inferir que existem filmes com altíssimos e baixíssimos valores de US Gross e Production Budegets.  

    Além disso, existem filmes com muitos votos e poucos votos, o que pode interferir em outras métricas como o IMDB Rating. Neste caso, por tratar-se de uma média de avalição, poucas votações podem gerar uma avaliação que não reflete a percepção geral do publico.